lib/interceptor.js

```diff
@@ -23,7 +24,27 @@ function Interceptor(scope, uri, method, requestBody, interceptorOptions) {
     this.scope = scope;
     this.interceptorMatchHeaders = [];
     this.method = method.toUpperCase();
+
+    // if URI includes a query string, separate it to use proper query matching
+    var queryIndex, queryStr;
```

Comment:
_Previously if the URI itself had a querystring, it behaved differently than if it was set via `query`. I don't see any good reason for this distinction._